doc/SpecifyingApplications/CreatingProfileInstanceList/CreatingProfileInstanceList.md

```diff
@@ -73,9 +115,8 @@ This is a step by step cookbook for creating the ProfileInstanceList.
 * The UUID shall follow the [Structure of UUIDs](../../ElementsApplicationPattern/Names/StructureOfUuids/StructureOfUuids.md).  
 * The LayerID shall be set on "integer".  
 * The ObjectType shall be set on "p" (Profile).  
-* The ApiSegment shall be set on "1" (management, which is individual to this application).  
-* The ApplicationNumber shall be set on "0" (the application itself).  
-* The SequenceNumber shall start at "00" and must be unique within the scope of instances of Profiles with the same LayerID.  
+* The ApiSegment shall be set on "mi" (management, which is individual to this application).  
```

Comment:
Same as mentioned in the action profile , 

This statement is contradicting with the concept in the Structure of uuid

Either the uuid-structure needs to be corrected 
Or
This line should be deleted

Reply:
Deleted this line.